design/one-pager-fn-claim-conditions.md

```diff
@@ -175,39 +252,5 @@ status:
     reason: Available
 ```
 
-### Team Implementation
-To accomplish this behavior, the team would need to configure their function
-pipeline to have the following behavior.
-1. The first step of the pipeline must always "reserve" the expected
```

Comment:
Note: I removed this part of the example as it only works if you have a single function in the pipeline.

If you have more than one function in the pipeline and the first one returns a fatal result, the remaining functions will not be called. So pre-loading conditions like this would be limited to single function pipelines unless we change the behavior, which seems like a pretty significant change IMO.